### DISCUSSION TITLE
Hello Everyone

### DISCUSSION THREAD

**Discussion by DanishIrfan1** [wagtail/gsoc#98] [General]
My name is **Danish Irfan** and I am currently a 6th-semester student pursuing a BS in Computer Science. I have a strong command of programming languages such as **C, C++, Python, and Django,** as well as some experience working with the **Wagtail CMS**. I am also an active contributor to open-source projects on GitHub and have uploaded several of my own projects to the platform.

When I first heard about the **GSoC** program, I was impressed by the opportunity to contribute to large-scale open-source projects while also gaining valuable experience and mentorship. As someone who is passionate about software development and eager to make a positive impact on the world, I am excited about the prospect of participating in this program and working with other like-minded individuals to tackle challenging projects and create innovative solutions.

**Comment by thibaudcolas** [comment]
Hi @DanishIrfan1, welcome! Most people have posted their introductions in https://github.com/wagtail/gsoc/discussions/1, which is where we asked everyone to post them, as well as gave guidelines on what to cover.
